microlith/grafana/provisioning/dashboards/couchbase-inventory.json

```diff
@@ -208,19 +208,28 @@
           {
             "matcher": {
               "id": "byName",
-              "options": "Host"
+              "options": "Cluster"
             },
             "properties": [
               {
                 "id": "custom.width",
                 "value": 185
+              },
+              {
+                "id": "links",
+                "value": [
+                  {
+                    "title": "Go to Couchbase UI",
+                    "url": "${__data.fields.Cluster}"
+                  }
+                ]
```

Comment:
We should probably enable "open in new tab" here, as it'll take over the Grafana tab otherwise (not enabled for the dashboard link because you'd expect that to take over)